frontend/src/components/Header/Header.tsx

```
@@ -1,16 +1,56 @@
   return (
-    <div className="flex justify-between max-w-2xl mx-auto">
-      <div className="order-2">
-        <ModeToggle />
+    <header className="shadow-md dark:bg-gray-800 pb-2">
+      <div className="px-4 py-3 flex justify-between items-center">
+        {/* Left Side: Project Name */}
+        <Link to="/" className="md:text-xl text-lg font-semibold font-serif">Project Sphere</Link>
+
+        {/* Center: Navigation Links (Hidden in Mobile) */}
+        <nav className="hidden md:flex space-x-6">
+          <a href="#" className="hover:text-gray-300">Home</a>
```

Biome (1.9.4)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)